Sources/WireGuardKitGo/in-tunnel-icmp_test.go

```diff
@@ -45,3 +48,47 @@ func TestIcmpSocketCloseFailsReadImmediately(t *testing.T) {
 		t.Fatalf("Expected the ICMP socket read to fail with an error, thus expected a negative erorr code, instead got %d", err)
 	}
 }
+
+// This test is disabled intentionally, since it relies on valid WireGuard keys
+// being set for both the client and the relay. It is left in the repo to allow
+// for easier manual testing.
+func testIcmpSocketParse(t *testing.T) {
+	privateKey, _ := base64.StdEncoding.DecodeString("mJiFq5mdExIZQVTt2QrL2o9sACkVAlUC7d/09+1wbkw=")
```

Comment:
Is this actually a valid key? Maybe you should replace it with some stub string.

Reply:
It is not a valid key anymore.
<!-- Reviewable comment -ODfv4VIEr7CzYPKJQbA:b-3nwz0c -->
<!-- Sent from Reviewable.io -->